### DISCUSSION TITLE
Erreur 500 - pb avec serveurs Ratp

### DISCUSSION THREAD

**Discussion by fablamenas** [pgrimaud/horaires-ratp-api#114] [General]
Il y a une petite error 500 depuis quelques semaines. Si c'est facile pour toi c'est cool. Bonne journée.

test: https://api-ratp.pierre-grimaud.fr/v4/destinations/metros/8

{
    "result": {
        "code": 500,
        "message": "Internal Server Error"
    },
    "_metadata": {
        "call": "GET /destinations/metros/8",
        "date": "2022-11-30T10:47:10+01:00",
        "version": 4
    }
}

**Comment by oudoulj** [comment]
https://github.com/pgrimaud/horaires-ratp-api/issues/112
